runtime/src/accounts_db.rs

```diff
@@ -1633,8 +1641,9 @@ impl AccountsDb {
         accounts_scan.stop();
 
         let mut clean_old_rooted = Measure::start("clean_old_roots");
+        let purges = [purges_in_root, purges_unrooted].concat();
         let (purged_account_slots, removed_accounts) =
-            self.clean_old_rooted_accounts(purges_in_root, max_clean_root);
+            self.clean_old_rooted_accounts(purges, max_clean_root);
```

Comment:
nits: how about renaming: `self.clean_accounts_older_than_root()` because we're now handling unrooted slots? more better naming is welcome :)

Reply:
Hah, yes totally agree!

Reply:
Done!